core/views.py

```diff
@@ -274,7 +274,7 @@ def get_queryset(self: "CommunityListView") -> models.QuerySet:
             super()
             .get_queryset()
             .prefetch_related("members")
-            .annotate(has_access=Exists(CommunityMember.objects.filter(community=OuterRef("pk"), user=user)))
+            .annotate(has_access=Exists(CommunityMember.objects.filter(community=OuterRef("pk"), user=user.id)))
```

Comment:
Nice fix!
From the look of the error I would probably add some conditions for unauthorized/authorized user, but this works great!
Approved.
Would be perfect though if you'd add a test for this case.